spec/classes/jenkins_repo_spec.rb

```diff
@@ -6,7 +8,7 @@
       let(:facts) { os_facts }
 
       describe 'repo' do
-        describe 'default' do
+        describe 'default' do # rubocop:todo RSpec/EmptyExampleGroup
```

Comment:
I believe this is https://github.com/rubocop/rubocop-rspec/issues/1244

Reply:
Just providing context. In the issue now there'll be a larger collection of examples.